CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
   unprivileged overlayfs.
 - Ensure bootstrap_history directory is populated with previous definition
   files, present in source containers used in a build.
+- Add e2e-test for unprivileged container build under different fakeroot modes.
```

Comment:
I don't think we need to mention to users about an additional e2e-test.  However, it would be good mention the new options.  I suggest
```
 - Add additional options to the build command for testing different fakeroot modes: `--userns`
   like the action flag and hidden options `--ignore-subuid`, `--ignore-fakeroot-command`,
   and `--ignore-userns`.
```

Reply:
https://github.com/apptainer/apptainer/pull/607#discussion_r944606257